credentialdigger/cli.py

```diff
@@ -610,14 +611,19 @@ def scan(self, repo_url, category=None, scanner=GitScanner,
             The id of the discoveries detected by the scanner (excluded the
             ones classified as false positives).
         """
-        def analyze_discoveries(model_manager, discoveries, verbose):
+        if debug is True:
+            logger.setLevel(level=logging.DEBUG)
+        else:
+            logger.setLevel(level=logging.INFO)
+            
+        def analyze_discoveries(model_manager, discoveries, debug):
             """ Use a model to analyze a list of discoveries. """
             false_positives = set()
 
             # Analyze all the discoveries ids with the current model
-            if verbose:
-                print('Analyzing discoveries with model %s' %
-                      model_manager.model)
+            if debug:
```

Comment:
Hi @marcorosa,
Thank you for the comments.
I believe that we can not remove this "if debug" since it is followed by a set of instructions that are more than a simple "print()".
If we are removing this "if debug", we have to decide whether to keep this block of code : 
```python
                for i in tqdm(range(len(discoveries))):
                    did = discoveries[i]
                    if model_manager.launch_model(self.get_discovery(did)):
                        false_positives.add(did)
```
which is giving a more detailed output. 
Or this : 
```python
                for did in discoveries:
                    if model_manager.launch_model(self.get_discovery(did)):
                        false_positives.add(did)
```

Thank you!